vault/init.go

```diff
@@ -386,6 +386,21 @@ func (c *Core) Initialize(ctx context.Context, initParams *InitParams) (*InitRes
 			}
 
 			results.RecoveryShares = recoveryUnsealKeys
+
+			keysToStore := [][]byte{barrierKey}
+			if err := c.seal.SetStoredKeys(ctx, keysToStore); err != nil {
```

Comment:
Why is this needed?

Reply:
Oops, it isn't. That was from one of many previous attempts.